test/schemas.test.js

```diff
@@ -9,7 +9,6 @@ describe('Forum user schema test', function() {
 
         f.validate(function(err) {
             expect(err.errors.username).to.exist;
-            expect(err.errors.displayName).to.exist;
```

Comment:
This should be validated to true for the test to pass

Reply:
Yeah, yup that's true. idk why I removed that.